gcbmgr

```diff
@@ -404,6 +404,12 @@ GCP_USER_TAG=${GCP_USER_TAG/@/-at-}
 # in the bucket name so convert . to -
 GCP_USER_TAG=${GCP_USER_TAG/\./-}
 
+if ((FLAGS_attended)); then
```

Comment:
Ca we call the flag `tail` or `follow`? I think those names are more commonly used elsewhere and are clearer to me, a non-native speaker.

Reply:
@hoegaarden -- Okay, will think of a better flag name! Let me get this in since the test-infra PR merged, so I can get an idea of the failures and then we'll rename in the next iteration.